### DISCUSSION TITLE
What is `serde` feature doing?

### DISCUSSION THREAD

**Discussion by omid** [jprochazk/garde#98] [Q&A]
What is `serde` feature doing?

**Comment by inklesspen** [comment]
It's used with the [Unvalidated](https://docs.rs/garde/latest/garde/validate/struct.Unvalidated.html) and [Valid](https://docs.rs/garde/latest/garde/validate/struct.Valid.html) wrapper types; if you enable the `serde` feature, then `Unvalidated` has `#[derive(Deserialize)]` and `#[serde(transparent)]` on it, so that you can deserialize directly into an `Unvalidated`.
